source/release-notes.txt

```diff
@@ -10,6 +10,48 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.20
+--------------
+
+*Released November, 2019*
+
+- New and improved :guilabel:`New Connection` user experience with
+  support for all connection options.
+
+- Added support for:
+
+  - Cloning :guilabel:`Favorites`.
```

Comment:
This is actually part of a larger effort which overhauls Favorite Connections in general. Also, "Favorites" is ambiguous because it could refer to either Favorite Connections or Favorite Queries. Afaik this is about Favorite Connections.

Suggest moving this to its own bullet and saying something like "Improved user experience for saving and sharing favorite connections", and link somewhere in there to the Favorite Connections page.